components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -356,13 +359,13 @@ func (p *pg) associatePolicyWithProjects(ctx context.Context,
 	// TODO this might be simplified as we modify how projects are assigned
 	// Drop any existing associations.
 	_, err := q.ExecContext(ctx,
-		`DELETE FROM iam_policy_projects WHERE policy_id=$1;`, policyID)
+		`DELETE FROM iam_policy_projects WHERE policy_id=policy_db_id($1);`, policyID)
```

Comment:
hrm that backtick and semicolon habit 
```suggestion
		"DELETE FROM iam_policy_projects WHERE policy_id=policy_db_id($1)", policyID)
```
but this really doesn't matter 😉

Reply:
it totally does-- on my list for later if time allows

Reply:
also one of them got mad when I tried it because a string was involved . . . hmmm can't recall